scripts/deploy/containerregistry.sh

```diff
@@ -82,6 +82,18 @@ fi
 
 sleep 10
 
+
+if ( echo ${DESTINATION_REGISTRY_HOST} |grep -q icr.io ); then 
+    echo "Updating skopeo configuration..."
+    mkdir codesign
+    echo "docker:" > codesign/default.yaml
+    echo "  tools.boomerangplatform.net:8500:" >> codesign/default.yaml
+    echo "    sigstore: https://$GLOBAL_REGISTRY_HOST/artifactory/boomeranglib-docker" >> codesign/default.yaml
+    echo "    sigstore-staging: file:///var/lib/atomic/sigstore" >> codesign/default.yaml
```

Comment:
@Amy0511 Rather than using echo commands a better approach is to use cat, e.g.
```
cat > codesign/default.yaml <<EOF
docker:
  tools.boomerangplatform.net:8500:
    sigstore: https://$GLOBAL_REGISTRY_HOST/artifactory/boomeranglib-docker
    sigstore-staging: file:///var/lib/atomic/sigstore
EOF
```